File: 1-easy/problems/1304-find-n-unique-integers-sum-up-to-zero/solution.ts
function sumZero(n: any): boolean | number | string | any {
  """
  :type n: int
  :rtype: List[int]
  """
  return [i for i in Array.from({length: -(n//2}, (_, i) => i), n//2+1) if not (i == 0 && n%2 == 0)];
}

export { sumZero };

File: 1-easy/problems/1304-find-n-unique-integers-sum-up-to-zero/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def sumZero(self, n):
        """
        :type n: int
        :rtype: List[int]
        """
        return [i for i in range(-(n//2), n//2+1) if not (i == 0 and n%2 == 0)]
